### PR TITLE
docs: Aries framework pkg godocs

### PR DESCRIPTION
**Pull request by rolsonquadras**
Part of #739 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: pkg/framework/aries/doc.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+/*
+Package aries provides access to the library consumers to create a framework with options. The framework
```
Comment (troyronda) [inline pkg/framework/aries/doc.go]: Would be an idea to update this text a bit to align with the README.

> A pluggable dependency framework, where implementors can customize primitives via Service Provider Interfaces (SPIs). We have a "batteries included" model where default primitives are included -- such as a key management system (KMS), crypto, data storage, digital hub integration, etc.

File: pkg/framework/aries/framework.go [hunk 2]
```diff
@@ -46,7 +46,8 @@ type Aries struct {
 // Option configures the framework.
 type Option func(opts *Aries) error
 
-// New initializes the Aries framework based on the set of options provided.
+// New initializes the Aries framework based on the set of options provided. This function returns a framework
+// which can be used to manage Aries clients by getting the framework context.
```
Comment (troyronda) [inline pkg/framework/aries/framework.go]: Should also create examples. See DID exchange godoc.

File: pkg/framework/aries/doc.go (hunk 1 above)
Comment (rolsonquadras) [reply pkg/framework/aries/doc.go]: [Updated](https://github.com/hyperledger/aries-framework-go/compare/1a0fabad4135a7495c46315752ad06c326249a89..6d8821c860b3f4c2f12b4e5048eacf470b90bcd2#diff-d8e0c85fc8df82bb6530d2620c04e3daR8).

File: pkg/framework/aries/framework.go (hunk 2 above)
Comment (rolsonquadras) [reply pkg/framework/aries/framework.go]: [Added example](https://github.com/hyperledger/aries-framework-go/compare/1a0fabad4135a7495c46315752ad06c326249a89..6d8821c860b3f4c2f12b4e5048eacf470b90bcd2#diff-6b0dce919a202a9ca9791c186a715b5cR17).

**Comment by rolsonquadras** [comment]
@sandrask I agree with both the points. Will update the example and push the changes.

**Pull request merged**